sortinghat/core/jobs.py

```diff
@@ -472,3 +476,16 @@ def _iter_split(iterator, size=None):
         slice_iter = itertools.islice(iterator, size)
         peek = next(slice_iter)
         yield itertools.chain([peek], slice_iter)
+
+
+def check_criteria(criteria):
+    """ Check if all given criteria are valid.
+
+    Raises an error if a criterion is not in the valid criteria list
+    (`email`, `name` and/or `username`).
+
+    :param criteria: list of criteria to check
+    """
+    valid_criteria = ['name', 'email', 'username']
+    if any(criterion not in valid_criteria for criterion in criteria):
+        raise ValueError(f"Invalid criteria {criteria}")
```

Comment:
To make it more useful for the users, wouldn't it be better to add to the error message the list of valid values?

Reply:
You're right, I've added it.